config/environments/test.rb

```diff
@@ -1,40 +1,44 @@
+# The test environment is used exclusively to run your application's
+# test suite. You never need to work with it otherwise. Remember that
+# your test database is "scratch space" for the test suite and is wiped
+# and recreated between test runs. Don't rely on the data there!
+
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
-
-  # The test environment is used exclusively to run your application's
-  # test suite. You never need to work with it otherwise. Remember that
-  # your test database is "scratch space" for the test suite and is wiped
-  # and recreated between test runs. Don't rely on the data there!
   config.cache_classes = false
```

Comment:
this option can be removed as has been replaced by `config.enable_reloading`

Reply:
@jsugarman Everything fails when I do that (see last commit)

Reply:
I've now reverted the change as everything broke (again) when I set this to true

Reply:
Oops, replied to the wrong comment above. [Put it here instead](https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/7628/commits/e938c579f41180fb0e0bac9cc506a999091b672a#r1967188836)